install/gcp/installer/util/gcloud.py

```diff
@@ -54,6 +54,16 @@ def get_gcloud_info():
     return project_id, authed_user, is_devshell
 
 
+def set_project_id(self):
```

Comment:
Would be better for this method to take in a project id instead of an the entire installer class